source/write/delete.txt

```diff
@@ -0,0 +1,150 @@
+.. _c-write-delete:
+
+================
+Delete Documents
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: remove, drop, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to remove
+documents from a MongoDB collection by performing **delete operations**.
+
+A delete operation removes one or more documents from a MongoDB collection.
+You can perform a delete operation by using the ``mongoc_collection_delete_one()``
+or ``mongoc_collection_delete_many()`` functions.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Delete Operations
+-----------------
+
+You can perform delete operations by using the following functions:
+
+- ``mongoc_collection_delete_one()``, which deletes *the first document* that matches
+  the search criteria
+- ``mongoc_collection_delete_many()``, which deletes *all documents* that match the
+  search criteria
+
+Each delete function accepts the following parameters:
+
+- Collection: Specifies the collection to update.
```

Comment:
[s] I think the parameter names in this list can be emphasized per the style guide: 
```suggestion
- **Collection**: Specifies the collection to update.
```